src/main/resources/create_db.sql

```diff
@@ -0,0 +1,63 @@
+CREATE DATABASE spring_application;
+
+CREATE TABLE clients
+(
+id int NOT NULL UNIQUE AUTO_INCREMENT,
+first_name varchar(50) NOT NULL,
+last_name varchar(50) NOT NULL,
+PRIMARY KEY (id),
+CHECK (id > 0)
+);
+
+CREATE TABLE products
+(
+id int NOT NULL UNIQUE AUTO_INCREMENT,
+price double NOT NULL,
+name varchar(50) NOT NULL,
+PRIMARY KEY (id),
+CONSTRAINT chk_product CHECK (id > 0 AND price > 0.0)
+);
+
+CREATE TABLE orders
```

Comment:
Orders and products related as 1 to 1, instead they should be related as 1 to N. Remove product_id from order table and add order_id to product table.